File: core/src/main/java/im/mingxi/miko/startup/ResStartUp.java
package im.mingxi.miko.startup;

import static im.mingxi.miko.startup.util.XRes.*;
import android.content.Context;
import im.mingxi.loader.hotpatch.HotPatch;

public class ResStartUp {
    public static void doLoad(Context ctx){
        addAssetsPath(ctx, HotPatch.INSTANCE.getHotPatchAPKPath());
    }
}

File: core/src/main/java/im/mingxi/miko/startup/ResStartUp.kt
package im.mingxi.miko.startup

import android.content.Context
import im.mingxi.loader.hotpatch.HotPatch.hotPatchAPKPath
import im.mingxi.loader.util.Constants
import im.mingxi.loader.util.PathUtil
import im.mingxi.miko.startup.util.XRes.addAssetsPath

object ResStartUp {
    fun doLoad(ctx: Context) {
        val apkPath = if (Constants.isHotPatch) hotPatchAPKPath else PathUtil.moduleApkPath
        addAssetsPath(ctx, apkPath)
    }
}